libs/tup-components/src/tickets/TicketDetailModal/TicketModal.global.css

```diff
@@ -133,3 +134,28 @@
   padding-right: 5rem;
   margin-left: 1rem;
 }
+
+.status-wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+
+.status-checkbox {
+  display: flex;
+  align-items: center;
+  margin-bottom: 0.75em;
+}
+
+.status-checkbox > input {
+  margin-right: 0.5em;
+}
+
+.button-hint {
+  display: flex;
+  flex-direction: column;
+}
+
+.button-hint > p {
+  margin-top: 1em;
+}
```

Comment:
**Excess Code.** I think `button-hint` is leftover. I do not see it used. Remove?